lib/std/Thread/Condition.zig

```diff
@@ -161,12 +161,27 @@ const WindowsImpl = struct {
             }
         }
 
-        const rc = os.windows.kernel32.SleepConditionVariableSRW(
-            &self.condition,
-            &mutex.impl.srwlock,
-            timeout_ms,
-            0, // the srwlock was assumed to acquired in exclusive mode not shared
-        );
+        const rc = blk: {
+            if (comptime builtin.mode == .Debug) {
```

Comment:
nitpick: could probably be repeated to avoid calling SleepConditionVariableSRW twice:
```
if (debug) store(0)
rc = SleepConditionVariableSRW(...)
if (debug) store(tid)
```